scripts/tiqit/__init__.py

```diff
@@ -44,8 +45,8 @@
 
 MAJ_VER   = 1
 MIN_VER   = 0
-PATCH_VER = 4
-DEV_VER   = 0
+PATCH_VER = 6
+DEV_VER   = 1
```

Comment:
DEV_VER shouldn't really be anything other than 0 in the master branch.

Reply:
Done